emscripten.py

```diff
@@ -920,6 +920,11 @@ def report_missing_symbols(all_implemented, pre):
       continue
     diagnostics.warning('undefined', 'undefined exported function: "%s"', requested)
 
+  # Handle main specially, unless IGNORE_MISSING_MAIN is set
+  if not shared.Settings.IGNORE_MISSING_MAIN:
+    if '_main' in shared.Settings.EXPORTED_FUNCTIONS and '_main' not in all_implemented:
+      exit_with_error('entry symbol not defined (pass --no-entry to suppress): main')
```

Comment:
why is this written as ending with `: main`? how about `"main" not defined (pass --no-entry to suppress)`? Or is this language modeled on a standard error format?

Reply:
This is the exact same format that wasm-ld will produce .. it means that error message is that same for fastcomp and the llvm backed.  I'll add a comment